src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working");

```

Comment:
The message in `System.out.println` should indicate the type of machine, which you have correctly done, but consider making it more informative. For example: 'Bulldozer is starting its work'.